tools/container/toolkit/toolkit.go

```diff
@@ -450,6 +452,12 @@ func installToolkitConfig(c *cli.Context, toolkitConfigPath string, nvidiaContai
 		"nvidia-container-runtime-hook.path":                nvidaContainerRuntimeHookPath,
 		"nvidia-container-runtime-hook.skip-mode-detection": opts.ContainerRuntimeHookSkipModeDetection,
 	}
+
+	toolkitRuntimeList := getNvidiaContainerRuntimeList(cfg, opts.ContainerRuntimeRuntimes.Value())
+	if len(toolkitRuntimeList) > 0 {
+		configValues["nvidia-container-runtime.runtimes"] = toolkitRuntimeList
+	}
```

Comment:
This should no longer be required.

We will ONLY set the binaries from the runtimes. Note that if `opts.ContainerRuntimeRuntimes` is set we will overwrite the value we set here as part of processing the "options" config options.

Reply:
> We will ONLY set the binaries from the runtimes.

I don't think we can do that. If you look at the default containerd config toml, the `BinaryName` fields are empty, meaning that the PATH-resolvable low-level-runtime binary paths will be used. So we still need the default list `["docker-runc", "runc", "crun"]`

Reply:
Could we implement it so that if the slice is empty we don't update the config. Alternatively see the comment below.